services/scan/src/store.test.ts

```diff
@@ -505,7 +507,11 @@ test('adjudication', () => {
   store.cleanupIncompleteBatches();
 });
 
-test('exportCvrs', () => {
+test('exportCvrs', async () => {
+  const buildCastVoteRecordMock = jest.spyOn(
+    buildCastVoteRecord,
+    'buildCastVoteRecord'
+  );
```

Comment:
This is fine for now, but mocking functions on a module namespace import won't work when we eventually move to using ES modules for real in Jest v28+. That's not happening soon, but consider this a nudge to use an alternative if one is readily available.